src/chart/gauge/GaugeView.ts

```diff
@@ -476,8 +477,8 @@ class GaugeView extends ChartView {
                 const itemModel = data.getItemModel<GaugeDataItemOption>(idx);
                 const emphasisModel = itemModel.getModel('emphasis');
                 if (showPointer) {
-                    const pointer = data.getItemGraphicEl(idx) as PointerPath;
-                    pointer.useStyle(data.getItemVisual(idx, 'style'));
+                    const pointer = data.getItemGraphicEl(idx) as Displayable;
+                    pointer.type !== 'image' && pointer.useStyle(data.getItemVisual(idx, 'style'));
```

Comment:
image also need to `useStyle` to clear style.

Here is a reference https://github.com/apache/echarts/blob/master/src/chart/helper/Symbol.ts#L275